src/main/kotlin/org/rust/ide/template/postfix/stringBasedPostfixTemplates.kt

```diff
@@ -100,6 +107,34 @@ class SomePostfixTemplate(provider: RsPostfixTemplateProvider) : SimpleExprPostf
 class OkPostfixTemplate(provider: RsPostfixTemplateProvider) : SimpleExprPostfixTemplate("ok", "Ok(expr)", provider)
 class ErrPostfixTemplate(provider: RsPostfixTemplateProvider) : SimpleExprPostfixTemplate("err", "Err(expr)", provider)
 
+class WrapTypePathPostfixTemplate(provider: RsPostfixTemplateProvider) : StringBasedPostfixTemplate("wrap", "\$wrapper$<path>", RsTypeParentsSelector(), provider) {
```

Comment:
`StringBasedPostfixTemplate` is not needed now, we can extend `PostfixTemplateWithExpressionSelector`